web/src/components/pages/WorkspaceList/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { AuthenticationRequiredPage } from "@reearth-cms/auth";
+import WorkspaceList from "@reearth-cms/components/organisms/Settings/workspaceList";
+
+export type Props = {
+  path?: string;
+  workspaceId?: string;
```

Comment:
I think you have this code mixed up between this Page and the Workspace Page

Reply:
Sorry I should have explained what I meant better. This is the WorkspaceList page, which doesn't receive workspaceId from the path. Whereas the Workspace page DOES receive the workspaceId from the path. Shouldn't these props be in the Workspace component, and this be empty?